mortar/src/main/java/mortar/MortarScope.java

```diff
@@ -169,9 +182,9 @@ public boolean isDestroyed() {
   }
 
   /**
-   * Sends {@link Scoped#onExitScope()} to all registrants and then clears the
-   * registration list. Recursively destroys all children. Parent scope drops its reference
-   * to this instance. Redundant calls to this method are safe.
+   * Sends {@link Scoped#onExitScope()} to all registrants and then clears the registration list
+   * and drops references to provided services. Parent scope drops its reference to this instance.
```

Comment:
Do we need/want to guarantee dropping reference services?  I'd kinda prefer not to.